release-team/role-handbooks/release-team-lead/README.md

```diff
@@ -56,15 +56,14 @@ In addition to the [Release Team Lead selection criteria][lead-criteria] you are
 Release Lead is a very time-consuming role, especially towards the end of the release cycle. Before you volunteer to be Release Lead, please make certain that your employer and your family are okay with you spending a lot of time on the release for the next three months. Here's a rough estimate of the time requirements by week:
```

Comment:
Not really to do with your PR but as you're in this area anyway:
```suggestion
Release Lead is a very time-consuming role, especially towards the end of the release cycle. Before you volunteer to be Release Lead, please make certain that your employer and your family are okay with you spending a lot of time on the release for the next four months. Here's a rough estimate of the time requirements by week:
```

Reply:
👍